android/src/main/kotlin/no/ntnu/beardblaster/LobbyRepository.kt

```diff
@@ -151,7 +151,7 @@ class LobbyRepository(private val db: FirebaseFirestore = Firebase.firestore) :
             db.collection(GAME_COLLECTION)
                 .document(id)
                 .update(
-                    "ended", LocalDateTime.now(ZoneOffset.UTC)
+                    "endedAt", LocalDateTime.now(ZoneOffset.UTC)
```

Comment:
Kan godt godta denne navneendringen! 

Bakgrunnen for denne var at når `(game.ended != null)` så er kampen slutt, selv om jeg er litt usikker på om den er i aktivt bruk p.t.